apps/storeutl.c

```diff
@@ -45,6 +46,8 @@ const OPTIONS storeutl_options[] = {
     {"keys", OPT_SEARCHFOR_KEYS, '-', "Search for keys only"},
     {"crls", OPT_SEARCHFOR_CRLS, '-', "Search for CRLs only"},
     {"subject", OPT_CRITERION_SUBJECT, 's', "Search by subject"},
+    {"multivalue-rdn", OPT_MULTIVALUE_RDN, '-',
+     "Enable support for multi-valued RDNs for -subject and -issuer arguments"},
```

Comment:
The placement between -subject and -issuer isn't the best.  Do you mind placing it last (after -r)?

Reply:
No problem, done.